packages/inter-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -1197,10 +1276,31 @@ export const prepareVaultManagerKit = (
               ),
           );
 
+          const [{ userSeatPromise, deposited }, liquidationRecorderKits] =
+            await Promise.all([
+              makeDeposit,
+              helper.makeLiquidationRecorderKits(timestamp),
+            ]);
+
+          /** @type PreAuctionState */
+          const preAuctionState = [...vaultData.entries()].map(
+            ([vault, data]) => [
+              `vault${vault.getVaultState().idInManager}`,
+              { ...data },
+            ],
+          );
+
           // This is expected to wait for the duration of the auction, which
           // is controlled by the auction parameters startFrequency, clockStep,
           // and the difference between startingRate and lowestRate.
-          const [proceeds] = await Promise.all([deposited, userSeatPromise]);
+          const [auctionSchedule, proceeds] = await Promise.all([
+            E(auctionPF).getSchedules(),
```

Comment:
At this point in time the auction is already finished. The question is whether or not we get the schedule where `liveScheduleAuction` is still pointing to the auction we just finished. This is what we need to indicate the `endTime` of the current liquidation auction.

I'm not sure if we can depend on `(E(acutionPF).getSchedule()).liveAuctionSchedule` is ___always___ going to point to right value due to distributed nature of the `vaultFactory` and `auctioneer` vats. Despite not being able put my finger directly on the possibility of a race condition, I still suggest we take the conservative path and call `E(acutionPF).getSchedule()` somewhere before await `deposited`.